sort_by_length.csv

```diff
@@ -0,0 +1,96 @@
+Score,Review length


```

Comment:
1. Put this in `/data`
2. Maybe better name if possible (`review_sorted_by_length`)